README.asciidoc

```diff
@@ -131,6 +134,13 @@ The plugin defines the following extension properties in the `docker` closure:
 |`apiVersion`    |String    |null                     |The remote API version. For most cases this can be left null.
 |=======
 
+[WARNING]
+====
+Default value of `docker.url` property is deprecated.
+It will be replaced to `unix:///var/run/docker.sock` in upcoming releases.
+Please adjust your build scripts accordingly.
```

Comment:
I agree with @eshepelyuk.  We should be coding to default docker behavior/configuration.  This will make it much easier for a docker newbie to start running and using this.  If he/she decides that further configuration is needed, then it should be done explicitly in docker and other dependent configurations.
Further, by specifying `localhost` you've limited this to running locally and not a remote machine anyway.  If that's the  case, then why not use a unix socket which is what docker is doing anyway rather than making more configuration changes.